### PR TITLE
Ghost roles listing in lobby tweak.

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

No more CCIAA or whatever roles listed there.

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: code/controllers/subsystems/ticker.dm [hunk 1]
```diff
@@ -444,16 +444,24 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	for(var/s in SSghostroles.spawners)
 		var/datum/ghostspawner/G = SSghostroles.spawners[s]
-		if(G.enabled && !("Antagonist" in G.tags) && !(G.loc_type == GS_LOC_ATOM && !length(G.spawn_atoms)))
+		if(G.enabled \
+			&& !("Antagonist" in G.tags) \
+			&& !(G.loc_type == GS_LOC_ATOM && !length(G.spawn_atoms)) \
+			&& (("Simple Mobs" in G.tags) || ("External" in G.tags)) \
+		)
```
Comment (Arrow768) [inline code/controllers/subsystems/ticker.dm:451]: Generally alright, but a new variable that controls if it should be visible in the lobby might be the better approach.

**Review by Arrow768 — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: code/controllers/subsystems/ticker.dm (hunk 1 above)
Comment (DreamySkrell) [reply code/controllers/subsystems/ticker.dm]: changed from `&& (("Simple Mobs" in G.tags) || ("External" in G.tags))`
into `&& (G.req_perms == null)`

Should be better like this.
I'd rather not be adding yet another variable.

**Review by JohnWildkins — APPROVED** [review]

**Comment by DreamySkrell** [comment]
tests failing cause random away sites error something ahhhhhhhhhh

**Pull request merged**